src/include/daos/tse.h

```diff
@@ -21,8 +21,14 @@
 #define TSE_TASK_SIZE		1024
 /* 8 bytes used for public members */
 #define TSE_PRIV_SIZE		1016
+
 /* tse_task arguments max length */
-#define TSE_TASK_ARG_LEN		880
+#ifdef __aarch64__
+/** pthread_mutex_t is 8 bytes more on aarch64 in obj_auxi_args->obj_reasb_req->orr_mutex */
+#define TSE_TASK_ARG_LEN	888
+#else
+#define TSE_TASK_ARG_LEN	880
+#endif
```

Comment:
I'm afraid only changing is not enough, we may ask the person who reported the bug to test it on arm node.
The reason is TSE_TASK_SIZE, TSE_PRIV_SIZE, TSE_TASK_ARG_LEN are calculated, and with such compiling assertion in tse.c -
D_CASSERT(sizeof(struct tse_task) == TSE_TASK_SIZE);                                                    
D_CASSERT(sizeof(struct tse_task_private) <= TSE_PRIV_SIZE);

If only enlarge TSE_TASK_ARG_LEN for arm, looks possibly break the original assumption.

Reply:
yea you're right we need to adjust those. let me fix that.

Reply:
fixed. also took Jeff's suggestion and extended it to the other variables.
i agree we need him to test this to verify as i mentioned in the ticket, but should get PR approval first.